gen-cpp/CppRacconChat.h

```diff
@@ -0,0 +1,18 @@
+#include "RacconChat.h"
+
+using namespace ::tutorial;
+
+class RacconChatHandler : virtual public RacconChatIf {
+ private:
```

Comment:
usually you define things in the following order:
public:
proteched:
private: